src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -225,154 +225,146 @@ public RowIdMatch mapValuesToRows(Snapshot snapshot,
             "WHERE R." + PDAO_ROW_ID_COLUMN + " = T." + PDAO_ROW_ID_COLUMN;
 
     @Override
-    public void createSnapshot(Snapshot snapshot, List<String> rowIds) {
+    public void createSnapshot(Snapshot snapshot, List<String> rowIds) throws InterruptedException {
```

Comment:
There are no actual code changes in this file other than adding InterruptedException to the method signatures and removing the try/catches that were previously catching InterruptedExceptions --right? I went through the diff and it looks like the rest is all formatting, but just wanted to make sure I didn't miss anything.

Reply:
Yes, that is all I did.